protocol/javatests/org/openyolo/protocol/CredentialTest.java

```diff
@@ -16,17 +16,18 @@
 
 package org.openyolo.protocol;
 
-import static org.assertj.core.api.Java6Assertions.assertThat;
-import static org.openyolo.protocol.AuthenticationMethods.EMAIL;
-
 import android.net.Uri;
 import android.os.Parcel;
+
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.robolectric.RobolectricTestRunner;
 import org.robolectric.annotation.Config;
 import org.valid4j.errors.RequireViolation;
 
+import static org.assertj.core.api.Java6Assertions.assertThat;
```

Comment:
One day, I will commit the Android Studio settings so this stops happening to me when I move machines...